creations/package.json

```diff
@@ -6,7 +6,9 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.1.3",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.1.0",
```

Comment:
@UmangLadha Whenever adding any dependency always use -s so that it gets saved in package json file.
I guess this was the error that day right ?